app/scripts/store/dimStoreItem.directive.js

```diff
@@ -59,7 +59,22 @@
         '    <div class="img" dim-bungie-image-fallback="::vm.item.icon" ng-click="vm.clicked(vm.item, $event)">',
         '    <div ng-if="vm.item.quality" class="item-stat item-quality" ng-style="vm.item.quality.min | qualityColor">{{ vm.item.quality.min }}%</div>',
         '    <img class="element" ng-if=":: vm.item.dmg && vm.item.dmg !== \'kinetic\'" ng-src="/images/{{::vm.item.dmg}}.png"/>',
+        '    <svg ng-if="vm.item.isNew" width="44" height="44" version="1.1" xmlns="http://www.w3.org/2000/svg">',
```

Comment:
Since this isn't dynamic, could it be moved into an external SVG and referenced as an `img`?